app/helpers/service_helpers.py

```diff
@@ -0,0 +1,4 @@
+def get_lot_name_from_acronym(blueprint, acronym):
+    """Return the full lot name for its acronym"""
+
+    return blueprint.config['LOTS'][acronym]
```

Comment:
The block in which it's called is in a `try` statement https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/add-service-page/app/main/views.py#L22, set up to catch KeyErrors.